app/assets/stylesheets/styles/components/guides/_new_guides.css.scss

```diff
@@ -103,6 +245,226 @@
+      width: 100%;
+      height: 100%;
+      transition: border .2s;
+
+      &:hover {
+        border-color: #CCCCCC;
+      }
+
+      p {
+        position: absolute;
+        bottom: 0;
+        font-weight: 600;
+        background: rgba(255,255,255,0.95);
+        border-radius: 3px;
+        padding: 2px 10px;
+      }
+    }
+
+    input:checked + .stage-option {
+      border: 3px solid $of-green;
+    }
+  }
+
+  /* Step 3 */
+  .stage-details {
+    section {
+      .row {
+        margin-bottom: 1.8rem;
+      }
+    }
+
+    // TODO: Refactor out into component CSS
+    /* Current stage breadcrumb nav */
+    .chosen-stages {
+      text-align: center;
+      margin: 1rem 0 2rem;
+
+      .chosen-stage {
+        display: inline-block;
+        width: 5rem;
+        cursor: pointer;
+        position: relative;
+
+        &::before {
+          content: '';
+          border-bottom: 2px solid $of-light;
+          height: 0;
+          width: 100%;
+          display: inline-block;
+          margin-left: 2rem;
+          margin-bottom: -2.25rem;
+          z-index: 0;
+          position: relative;
+        }
+
+        &:last-child::before {
+          display: none;
+        }
+
+        .bubble {
+          width: 3.5rem;
+          height: 3.5rem;
+          border-radius: 10rem;
+          background-origin: padding-box;
+          background-size: 100%;
+          border: 2px solid $of-yellow;
+          margin: 0 auto;
+          transition: border-color .2s;
+          position: relative;
+          z-index: 1;
+
+          &:before {
+            font-size: 2rem;
+            position: relative;
+            color: $of-green;
+            width: 100%;
+            height: 100%;
+            display: block;
+            padding-top: .75rem;
+            background-color: rgba(255,255,255,.7);
+            border-radius: 10rem;
+          }
+          &.fa-check {
+            border: 2px solid $of-green;
+          }
+        }
+
+        &:last-child {
+          margin-right: 0;
+        }
+        &.editing .bubble {
+          border-color: $of-green;
+        }
+        p {
+          text-align: center;
+          opacity: 0;
+          transition: opacity .2s;
+          width: 120px;
+          margin-left: -20px;
+        }
+        &.editing p {
+          opacity: 1;
+        }
+
+      }
+    }
+
+    /* each step has actions to fill out */
+    .actions {
+
+       ul {
+        margin: 0;
+        list-style: none;
+
+        li {
+          padding: .25rem .5rem;
+          cursor: pointer;
+          transition: 0.5s;
+          margin: .25rem 0;
+          color: #666666;
+          border: 2px solid $secondary-color;
+          border-radius: $global-radius;
+        }
+
+        li:hover, li.selected {
+          border-color: $of-green;
+          color: $of-green;
+        }
+
+        li.completed {
+          border-color: $of-green;
+          background: $of-green;
+          color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.